src/main/java/com/github/princesslana/eriscasper/util/FormatType.java

```diff
@@ -0,0 +1,27 @@
+package com.github.princesslana.eriscasper.util;
+
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.OutputStream;
+import javax.imageio.ImageIO;
+
+public enum FormatType {
+  GIF("gif"),
+  JPG("jpg"),
+  JPEG("jpeg"),
```

Comment:
So, you said ImageIO supports both. What is the difference between them?

Reply:
not sure lemme look into it